config.js

```diff
@@ -0,0 +1,48 @@
+require('dotenv').config()
+
+const env = process.env.NODE_ENV // 'dev' or 'test'
+
+const dev = {
+  app: {
+    port: 3000
+  },
+  db: {
+    host: 'localhost',
+    port: 5432,
+    user: 'postgress',
+    name: 'phantom_dev',
+    dialect: 'postgress',
+    pool: {
+      max: 5,
+      min: 0,
+      acquire: 30000,
+      idle: 10000
+    }
+  }
+}
+
+const test = {
+  app: {
+    port: 8080
+  },
+  db: {
+    host: 'localhost',
+    port: 27017,
+    user: 'postgress',
+    name: 'phantom_test',
```

Comment:
The same here, `host`, `port`, `user` and `name` should be variables that need to be added in ```.env```

Reply:
Changed the structure of the config.js and created its own folder **config**. all these changes are depicted in images below.
<img src="https://i.ibb.co/DKP5nvX/report.png" alt="report" border="0">
<img src="https://i.ibb.co/kg5g89J/report1.png" alt="report1" border="0">